honda_crv_2017_can.dbc

```diff
@@ -2,255 +2,306 @@ VERSION ""
- SG_ SET_ME_X00 : 22|7@0+ (1,0) [0|127] ""  EPS
- SG_ SET_ME_X00_2 : 31|8@0+ (1,0) [0|0] ""  EPS
- SG_ CHECKSUM : 39|4@0+ (1,0) [0|15] ""  EPS
- SG_ COUNTER : 33|2@0+ (1,0) [0|3] ""  EPS
-
-BO_ 232 BRAKE_COMMAND: 7 NEO
- SG_ CAR_SPEED : 3|10@0- (1,0) [1|0] "" NEO
- SG_ XXX : 8|1@0- (1,0) [1|0] "" XXX
- SG_ COMPUTER_BRAKE_REQUEST : 29|1@0+ (1,0) [0|0] "" EBCM
- SG_ COMPUTER_BRAKE : 39|16@0+ (1,0) [0|0] "" EBCM
- SG_ COUNTER : 53|2@0+ (1,0) [0|3] ""  BDY
- SG_ CHECKSUM : 51|4@0+ (1,0) [0|3] ""  BDY
+ SG_ STEER_TORQUE : 7|16@0- (1,0) [-3840|3840] "" EPS
+ SG_ STEER_TORQUE_REQUEST : 23|1@0+ (1,0) [0|1] "" EPS
+ SG_ SET_ME_X00 : 22|7@0+ (1,0) [0|127] "" EPS
+ SG_ SET_ME_X00_2 : 31|8@0+ (1,0) [0|0] "" EPS
+ SG_ COUNTER : 33|2@0+ (1,0) [0|3] "" EPS
+ SG_ CHECKSUM : 39|4@0+ (1,0) [0|15] "" EPS
+
+BO_ 232 BRAKE_HOLD: 7 XXX
+ SG_ XMISSION_SPEED3 : 7|14@0- (1,0) [1|0] "" XXX
+ SG_ COMPUTER_BRAKE : 39|16@0+ (1,0) [0|0] "" XXX
+ SG_ COUNTER : 53|2@0+ (1,0) [0|3] "" XXX
+ SG_ CHECKSUM : 51|4@0+ (1,0) [0|3] "" XXX
+ SG_ COMPUTER_BRAKE_REQUEST : 29|1@0+ (1,0) [0|0] "" XXX
+ SG_ SET_TO_1 : 9|2@0- (1,0) [1|0] "" XXX
+ SG_ ZEROS_BOH2 : 28|5@0+ (1,0) [0|31] "" XXX
 
 BO_ 304 GAS_PEDAL: 8 PCM
- SG_ ENGINE_TORQUE_ESTIMATE : 7|16@0- (1,0) [-1000|1000] "Nm"  NEO
- SG_ ENGINE_TORQUE_REQUEST : 23|16@0- (1,0) [-1000|1000] "Nm"  NEO
- SG_ CAR_GAS : 39|8@0+ (1,0) [0|255] ""  NEO
- SG_ COUNTER : 53|2@0+ (1,0) [0|3] ""  NEO
- SG_ CHECKSUM : 51|4@0+ (1,0) [0|3] ""  NEO
+ SG_ ENGINE_TORQUE_ESTIMATE : 7|16@0- (1,0) [-1000|1000] "Nm" NEO
+ SG_ ENGINE_TORQUE_REQUEST : 23|16@0- (1,0) [-1000|1000] "Nm" NEO
+ SG_ CAR_GAS : 39|8@0+ (1,0) [0|255] "" NEO
+ SG_ COUNTER : 53|2@0+ (1,0) [0|3] "" NEO
+ SG_ CHECKSUM : 51|4@0+ (1,0) [0|3] "" NEO
 
 BO_ 330 STEERING_SENSORS: 8 EPS
- SG_ STEER_ANGLE : 7|16@0- (-0.1,0) [-500|500] "deg"  NEO
- SG_ STEER_ANGLE_RATE : 23|16@0- (-1,0) [-3000|3000] "deg/s"  NEO
- SG_ STEER_ANGLE_OFFSET : 39|8@0- (-0.1,0) [-128|127] "deg"  NEO
- SG_ STEER_WHEEL_ANGLE : 47|16@0- (-0.1,0) [-500|500] "deg"  NEO
- SG_ COUNTER : 61|2@0+ (1,0) [0|3] ""  NEO
- SG_ CHECKSUM : 59|4@0+ (1,0) [0|3] ""  NEO
+ SG_ STEER_ANGLE : 7|16@0- (-0.1,0) [-500|500] "deg" NEO
+ SG_ STEER_ANGLE_RATE : 23|16@0- (-1,0) [-3000|3000] "deg/s" NEO
+ SG_ STEER_ANGLE_OFFSET : 39|8@0- (-0.1,0) [-128|127] "deg" NEO
+ SG_ STEER_WHEEL_ANGLE : 47|16@0- (-0.1,0) [-500|500] "deg" NEO
+ SG_ COUNTER : 61|2@0+ (1,0) [0|3] "" NEO
+ SG_ CHECKSUM : 59|4@0+ (1,0) [0|3] "" NEO
 
 BO_ 340 XXX_4: 8 XXX
 
 BO_ 344 POWERTRAIN_DATA: 8 PCM
- SG_ TRANSMISSION_SPEED : 7|16@0+ (0.002759506,0) [0|70] "m/s"  NEO
- SG_ ENGINE_RPM : 23|16@0+ (1,0) [0|15000] "rpm"  NEO
- SG_ TRANSMISSION_SPEED2 : 39|16@0+ (0.002759506,0) [0|70] "m/s"  NEO
- SG_ COUNTER : 61|2@0+ (1,0) [0|3] ""  NEO
- SG_ CHECKSUM : 59|4@0+ (1,0) [0|3] ""  NEO
+ SG_ XMISSION_SPEED : 7|16@0+ (0.002759506,0) [0|70] "m/s" NEO
+ SG_ ENGINE_RPM : 23|16@0+ (1,0) [0|15000] "rpm" NEO
+ SG_ XMISSION_SPEED2 : 39|16@0+ (0.1723,0) [1|0] "" NEO
```

Comment:
What do you think the unit is here? not m/s anymore? What about the limits?

Reply:
I'll revert that to what we have in the other Hondas, I was doing something else with it